scripts/build.sh

```diff
@@ -139,9 +139,9 @@ function Test {
   pushd "$TestDir"
 
   if [ -z "$remaining" ]; then
-    ctest --build-config "$configuration" --output-on-failure
+    ctest --build-config "$configuration" --output-on-failure --VV
   else
-    ctest --build-config "$configuration" --output-on-failure "${remaining[@]}"
+    ctest --build-config "$configuration" --output-on-failure --VV "${remaining[@]}"
```

Comment:
Is this still required?

Reply:
No idea - I don't even remember what it does. :^)